scripts/create_fixtures

```diff
@@ -21,8 +21,8 @@ main() {
     rm -rf "$schemes_repo_dir"
   fi
 
-  if [ -d "$fixtures_path" ]; then
-    rm -r "$fixtures_path"
+  if [ -f "$schemes_file_path" ]; then
+    rm -f "$schemes_file_path"
```

Comment:
Delete only schemes file (`schemes.txt`) instead of entire fixture directory. This is due to that configs for tests, e.g., [`base16-dracula.json`](https://github.com/tinted-theming/tinty/blob/main/fixtures/base16-dracula.json), are adding to fixture directory, too.